### PR TITLE
[BEAM-739] Log exceptions full stack track in IT tests

### PR DESCRIPTION
**Pull request by peihe**
R: @lukecwik @jasonkuster

### CODE REVIEW

**Comment by lukecwik** [comment]
Isn't there something we can configure on maven surefire to output the entire stack trace when the test fails instead of doing it by hand?

File: examples/java/src/test/java/org/apache/beam/examples/WordCountIT.java [hunk 1]
```diff
@@ -64,6 +67,13 @@ public void testE2EWordCount() throws Exception {
     String e2eTestInputPath = "gs://apache-beam-samples/apache/LICENSE";
     options.setInputFile(e2eTestInputPath);
 
-    WordCount.main(TestPipeline.convertToArgs(options));
+    try {
+      WordCount.main(TestPipeline.convertToArgs(options));
+    } catch (Exception e) {
+      for (StackTraceElement elem : e.getStackTrace()) {
```
Comment (lukecwik) [inline examples/java/src/test/java/org/apache/beam/examples/WordCountIT.java]: This is equivalent to LOG.error("Failed to process WordCountIT:", e);

See: http://slf4j.org/faq.html#paramException

Comment (peihe) [inline examples/java/src/test/java/org/apache/beam/examples/WordCountIT.java]: removed

**Comment by peihe** [comment]
turned off trimStackTrace in maven-failsafe-plugin
(mavens-surefire-plugin is already off)

**Comment by lukecwik** [comment]
Please remove the now duplicate configuration in:
sdks/java/io/kinesis/pom.xml
examples/java/pom.xml
sdks/java/io/google-cloud-platform/pom.xml

**Comment by peihe** [comment]
removed the duplicated config property
PTAL

**Comment by lukecwik** [comment]
LGTM, merging

**Pull request closed without merging**